compiler/rustc_mir_transform/src/instsimplify.rs

```diff
@@ -147,6 +157,30 @@ impl<'tcx> InstSimplifyContext<'tcx, '_> {
         }
     }
 
+    /// Transform "Aggregate(RawPtr, \[p, ()\])" ==> "Cast(PtrToPtr, p)".
```

Comment:
This escaping is really unfortunate. Is it possible to use backticks for code formatting instead?